### DISCUSSION TITLE
Status?

### DISCUSSION THREAD

**Discussion by jvinkovic** [MapsterMapper/Mapster#695] [General]
There are no changes for quite some time. Is development ceasing? 
Is there a need to help with development or?
@satano any comment?

**Comment by satano** [comment]
Well, I do not know. I am not maintainer of this project, I just made a couple of pull request some time ago.

**Reply by Skyppid** [reply]
@satano Who else maintains it then? When looking at the MapsterMapper organization you're the only person in there? I also feel like there's an issue and it's a severe problem that no one seems to feel responsible to check PRs.

Who owns this repository? I think the community would gladly help, it's widely used. I really would hate if I had to fork it to fix issues and deploy my own package to use it ...

If you don't feel responsible how about making a discussion were some of the community apply to overtake the ownership or add them as maintainer at least? This library is too good and widely apapted to now get abandoned.

And I really need a fix in there as this is blocking our development right now :(

**Reply by satano** [reply]
I am just an ordinary member of the MapsterMapper organization. There are several owners: @andrerav, @chaowlert, @eswann, @FariborzRazban, @ningenang. My last PR was merged by @andrerav, all PRs before by @chaowlert.

So guys, is here anybody still maintaining the Mapster? Thank you very much.

**Reply by Arcalise08** [reply]
https://github.com/MapsterMapper/Mapster/discussions/713
